mount/titus-mount-container-to-container.c

```diff
@@ -91,9 +91,14 @@ static int setup_fsfd_in_namespaces(int sk, int src_nsfd, const char *src_path)
 		return 1;
 	}
 
+	setuid(0);
+	setgid(0);
```

Comment:
Should we check for an error here?

Reply:
Ehhh.
I'm not 100% sure.
The whole point of the mkdir thing is in theory to be `mkdir || true`.
I don't really care about errors, *unless* it actually makes something not work.
I'll skip them on these lines for now.